app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::UsersController < ApplicationController
+  def create
+    begin
+      user_data = JSON.parse(request.body.string)
+      User.create!(user_data)
+      render status: 204
```

Comment:
I'll take this as a the general way we want to handle the return of most post requests etc.

Reply:
Yep!
I figured some explicit exception handling would be best :tada: